### ISSUE TITLE
Could not install packages due to an EnvironmentError: [Errno 13]

### ISSUE THREAD

**Issue by MrE4GLe90** [MatthewPierson/Vieux#209]
hello 
lm trying to downgrade iPad Air cellular from iOS 12.4.8 to iOS 10.3.3
l followed the guide after typing pip3 install -r requirements.txt seems there is an error, l tried both fixes 1 and 2
<img width="590" alt="Screen Shot 1442-03-28 at 5 29 03 AM" src="https://user-images.githubusercontent.com/74440061/99137358-5a02f000-263b-11eb-9f9e-152dd5811b1e.png">

lm using hackintosh Catalina 10.15.7 i5 3570k 16gb ram

**Comment by MrE4GLe90** [comment]
I succeeded adding "--user"
After pip3 command, command will be like this:

pip3 install -r requirements.txt --usr

Now my iPad Air is downgraded from ios 12.4.8 to 10.3.3

**Comment by RolphNL** [comment]
I am having an iPad Air but my issue is "unsupported model"

Downloaded my IPSW from the ipsw.me website. Where did you get yours from?

**Comment by MrE4GLe90** [comment]
> I am having an iPad Air but my issue is "unsupported model"
> 
> 
> 
> Downloaded my IPSW from the ipsw.me website. Where did you get yours from?

The same site, make sure you download the right model

My iPad Air is cellular
Model: A1475

Good luck

**Comment by RolphNL** [comment]
> > I am having an iPad Air but my issue is "unsupported model"
> > Downloaded my IPSW from the ipsw.me website. Where did you get yours from?
> 
> The same site, make sure you download the right model
> 
> My iPad Air is cellular
> Model: A1475
> 
> Good luck

Yes I think I did, I am having the Cellular iPad Air as well. 4.2
Downloaded the link like 5 different times.

But keep on coming with the same issue, unsupported model
